UnityGsdk/Assets/Tests/GSDKTests.cs

```diff
@@ -0,0 +1,413 @@
+using System;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+using PlayFab;
+using PlayFab.MultiplayerAgent.Model;
+using PlayFab.MultiplayerAgent.Helpers;
+using static UnityEditor.Experimental.GraphView.GraphView;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class GSDKTests
+{
+    private readonly string ConfigFilePath = Application.dataPath + "/Tests/testConfig.json";
+
+    private static SimpleJsonInstance _jsonInstance = new SimpleJsonInstance();
+
+    private GSDKConfiguration _testConfig;
+
+    [SetUp]
+    public void Setup()
```

Comment:
do we expect customers to include these tests when they use GSDK? 
(to be clear, this can be totally fine, just curious)

Reply:
This could easily go either way, but I wasn't necessarily writing these to serve a customer scenario, only for pipeline unit testing.

Reply:
Gotcha, we should think if we need to update the “install GSDK” docs to tell the customers that they don’t need these files (or something like that)